markdown/plext/using_pljava.html.md.erb

```diff
@@ -255,33 +272,48 @@ This section provides information about writing functions with PL/Java.
 - [Returning a SETOF \<scalar type\>](#returnsetofscalar)
 - [Returning a SETOF \<complex type\>](#returnsetofcomplex)
 
+You must have a Java SDK installed to write Java methods.
 
 ### <a id="sqldeclaration"></a>SQL Declaration 
+PL/Java functions are defined using the standard SQL [CREATE FUNCTION](../reference/sql/CREATE-FUNCTION.html) syntax. A PL/Java user-defined function specifies `LANGUAGE java` (trusted) or `LANGUAGE javau` (untrusted). The function must also include `RETURNS` and `AS` clauses.
 
-A Java function is declared with the name of a class and a static method on that class. The class will be resolved using the classpath that has been defined for the schema where the function is declared. If no classpath has been defined for that schema, the public schema is used. If no classpath is found there either, the class is resolved using the system classloader.
+The `RETURNS` clause identifies the type of the function's return value. 
 
-The following function can be declared to access the static method getProperty on `java.lang.System` class:
+Use the `AS` clause to identify the name of a Java class and the static method on that class that will be invoked when the PL/Java UDF is executed. The arguments to the UDF will be passed to the identified Java method.
```

Comment:
Minor edit:  It's almost always preferable to write in the present tense.  Easier to parse and usually fewer words involved.  So in this case, maybe:  "Use the `AS` clause to identify the name of the Java class and the static method on that class that HAWQ invokes when it executes the function. Any arguments to the UDF are also passed to that method."